packages/frontend/src/components/common/modal/ChartCreateModal.tsx

```diff
@@ -23,34 +23,45 @@ import {
 interface ChartCreateModalProps extends DialogProps {
     savedData: CreateSavedChartVersion;
     onClose?: () => void;
+    spaceUuid?: string;
```

Comment:
suggestion: I would have called this `defaultSpaceUuid` and kept the `spaceUuid` state. 

When reading the code, I know that `defaultSpaceUuid` is not a safe value to use. While `spaceUuid` sounds like a set in stone value.

Reply:
applied the same rule to `DashboardCreateModal.tsx` for consistency